cmd/common/snapshot/process_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snapshot
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/snapshot/process/types"
+)
+
+type stsTestCase struct {
+	val      uint64
+	valMax   uint64
+	width    uint64
+	expected string
+}
+
+func TestCreateTree(t *testing.T) {
```

Comment:
Maybe add another test where `createTree` cannot find the root event, something where all processes are children of processes in the list.